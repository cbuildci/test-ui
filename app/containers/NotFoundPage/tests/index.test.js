import React from 'react';
import { snapshots } from '../../../../internals/testing/snapshot-util';
import NotFound from '../index';

describe('<NotFound />', () => {
    it('should render the expected JSX', () => {
        snapshots(
            <NotFound/>
        );
    });
});
