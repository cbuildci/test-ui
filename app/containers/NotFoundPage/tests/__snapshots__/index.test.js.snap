// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NotFound /> should render the expected JSX 1`] = `
<React.Fragment>
  <HelmetWrapper
    defer={true}
    encodeSpecialCharacters={true}
  >
    <InjectIntl(FormattedTitle)
      defaultMessage="Page Not Found"
      id="cbuildci.containers.NotFoundPage.title"
    />
  </HelmetWrapper>
  <PageHeader>
    <FormattedMessage
      defaultMessage="Page Not Found"
      id="cbuildci.containers.NotFoundPage.header"
      values={Object {}}
    />
  </PageHeader>
  <p>
    <FormattedMessage
      defaultMessage="The requested page does not exist."
      id="cbuildci.containers.NotFoundPage.bodyMessage"
      values={Object {}}
    />
  </p>
</React.Fragment>
`;

exports[`<NotFound /> should render the expected JSX 2`] = `
Array [
  <h1
    class="mt-3 pb-2"
    style="padding-bottom:8px;border-bottom:1px solid #CCC"
  >
    <span>
      Page Not Found
    </span>
  </h1>,
  <p>
    <span>
      The requested page does not exist.
    </span>
  </p>,
]
`;
